hello.html

```diff
@@ -13,50 +13,56 @@
     </head>
   	<body>
       <header>
-        <div id="container">
-	    <img class="logo" src="https://engineering.dartmouth.edu/styleguide/web/png/shield/ThayerShield_Green.png">
-	    <h1 class="main_heading"> The Content Teacher </h1>
-	      </div>
+        <div class="container">
+	       <img class="logo" src="https://engineering.dartmouth.edu/styleguide/web/png/shield/ThayerShield_Green.png">
+	       <h1 class="main_heading"> The Content Teacher </h1>
+        </div>
    	  </header>
    	  <nav>
-    		<button class="dropbtn" onclick="location.href='hello.html'">Home</button>
-    		<div class="dropdown">
-   	    	<button class="dropbtn">Information</button>
+        <div class="container">
+          <ul>
+    		  <li id="active" class="dropbtn" onclick="location.href='hello.html'">Home</li>
+    		  <div class="dropdown">
```

Comment:
Donot incude 'div' tag in between 'li' tags. Always include such tags inside 'li' tags.